drivers/wdm/audio/hdaudbus_new/buspdo.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS Kernel Streaming
+ * FILE:            drivers/wdm/audio/hdaudbus/buspdo.h
+ * PURPOSE:         HDAUDBUS Driver
+ * PROGRAMMER:      Coolstar TODO
+                    Johannes Anderwald
+ */
+
```

Comment:
- Use new style file header
- Some files lack a file header
- Do we have Coolstar's real name?

Reply:
Why do you think we break the rules by accepting a code from the contributor only because he did not provide his real name and surname? Because you don't trust them personally? Or because it's a first-time contributor? I don't see any valid point in your comment.
Do our rules complain we necessarily need to know the real name and surname of the contributor? If so, we probably can simply ask him to provide it to us. It's not a problem I guess.

Reply:
> This rule also applies for 3rd-party code of single individuals. 


We have a history of importing anonymous 3rd party code before actually. So I’m a little unsure how true this is. 

On another hand this got signed by MS… so….. just pull the cert if we care that much.

Reply:
This person has been credited in many places under the same nickname, so you can use it: https://github.com/search?q=coolstarorganization%40gmail.com&type=code